<launch>
  <node name="ronex_ctrl_spawner"
	pkg="sr_ronex_controllers" type="load_passthrough_controllers.py" output="screen"/>
</launch>
